sample-cap/srv/ai-api/ai-api-service.cds

```diff
@@ -0,0 +1,5 @@
+@path: 'ai-api'
+service AiApiService {
+  // entity Deployments as projection on AiDeploymentsEntity;
+  action getDeployments() returns String;
```

Comment:
This claims to return String, but the implementation actually `return response.resources;` which should be a list, right?

Reply:
I get your point. Technically, we should define an `AiDeployment` cds entity as well. But... a bit too complicated for this simple sample. I now changed it to `array of String`, which is also not totally correct, but maybe it helps.

Reply:
Done. Now it is also technically correct with stringify 😆